frontend/public/components/_dropdown.scss

```diff
@@ -1,7 +1,6 @@
-$color-dropdown-hover: rgba(0, 0, 0, .05);
-$color-bookmarker--hover: #AAA;
-$color-bookmarker: #DDD;
-$dropdown-link-hover-border: 1px solid #bee1f4;
+$color-bookmarker--hover: #999;
```

Comment:
Nit:  here and next line:  use pf color vars?  Maybe `$color-pf-black-500` and `$color-pf-black-400`?